homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,90 @@
+"""Support for Lutron lights."""
+import logging
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, Light
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Setup Lutron lights."""
+    area_devs = {}
+    devs = []
+    for (area_name, device) in LUTRON_DEVICES['light']:
+        dev = LutronLight(hass, area_name, device, LUTRON_CONTROLLER)
+        area_devs.setdefault(area_name, []).append(dev)
+        devs.append(dev)
+    add_devices_callback(devs)
+
+    for area in area_devs:
+        if area not in LUTRON_GROUPS:
+            continue
+        gr = LUTRON_GROUPS[area]
+        gr.update_tracked_entity_ids(
+            list(gr.tracking) + [dev.entity_id for dev in area_devs[area]])
+
+
+def to_lutron_level(level):
+    """Convert the given HASS light level (0-255) to Lutron (0.0-100.0)."""
+    return float((level * 100) / 255)
+
+
+def to_hass_level(level):
+    """Convert the given Lutron (0.0-100.0) light level to HASS (0-255)."""
+    return int((level * 255) / 100)
+
+
+class LutronLight(LutronDevice, Light):
+    """Representation of a Lutron Light, including dimmable."""
+
+    def __init__(self, hass, area_name, lutron_device, controller):
+        """Initialize the light."""
+        self._prev_brightness = None
+        LutronDevice.__init__(self, hass, area_name, lutron_device, controller)
+
+    @property
+    def brightness(self):
+        """Return the brightness of the light."""
+        new_brightness = to_hass_level(self._lutron_device.last_level())
```

Comment:
Is `last_level()` requesting data or is it returning a cached value?

Asking because code that returns a property should not make requests.